src/apps/trainings/services/bayesian_elo.py

```diff
@@ -54,17 +54,23 @@ def update_ratings_iteratively(self, number_of_iterations):
                 self._update_specific_network_log_gamma(network_id, network_log_gamma)
             self._reset_anchor_log_gamma()
 
+        # Count games by network
+        df = pandas.DataFrame()
+        df["reference_network"] = self._detailed_tournament_results["reference_network"]
+        df["nb_games"] = 0.0
+        df["nb_games"] += self._detailed_tournament_results["total_games_white"]
+        df["nb_games"] += self._detailed_tournament_results["total_games_black"]
```

Comment:
I'm rusty with panda but is `new_df["col1"] = old_df["col1"]` keeping the index of old_df (so that network are sumed, even if the order differs ?

Or does it really on having everything in the same order ?

Reply:
I'm not entirely sure what you're asking. Can you explain what specifically you are worried about with the current code that might have a bug? Which of the indices are you worried about a mismatch here in this Elo game count computation?